demos/projects/ESPRESSIF/aziotkit/README.md

```diff
@@ -0,0 +1,305 @@
+  Have a [logical device](https://docs.microsoft.com/azure/iot-hub/iot-hub-create-through-portal#register-a-new-device-in-the-iot-hub) created in your Azure IoT Hub using your preferred authentication method* | Have an [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/how-to-manage-enrollments#create-a-device-enrollment) created in your instance of DPS using your preferred authentication method*
+
+  *While this sample supports SAS keys and Certificates, this guide will refer only to SAS keys.
+
+## Install prerequisites
+
+1. GIT
+
+    Install `git` following the [official website](https://git-scm.com/).
+
+2. ESP-IDF
+
+    Install the [Espressif ESP-IDF](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/index.html#get-started) following Espressif official documentation (steps 1 to 4).
+
+3. Azure IoT Embedded middleware for FreeRTOS
+
+
+    Clone the following repo to download all sample device code, setup scripts, and offline versions of the documentation.
+
+    **If you previously cloned this repo in another sample, you don't need to do it again.**
+
+    ```bash
+    git clone https://github.com/Azure-Samples/iot-middleware-freertos-samples.git
+    ```
+
+    To initialize the repo, run the following commands:
+
+    ```bash
+    cd iot-middleware-freertos-samples
+    git submodule update --init --recursive --depth 1
+    ```
+
+  You may also need to enable long path support for both Microsoft Windows and git:
+  * Windows: <https://docs.microsoft.com/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd#enable-long-paths-in-windows-10-version-1607-and-later>
+  * Git: as Administrator run `git config --system core.longpaths true`
+
+
+## Prepare the sample
+
+To connect the ESPRESSIF ESP32 to Azure, you will update the sample configuration, build the image, and flash the image to the device.
+
+### Update sample configuration
+
+The configuration of the ESPRESSIF ESP32 sample uses ESP-IDF' samples standard [kconfig](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/kconfig.html) configuration.
+
+On a [console with ESP-IDF](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/index.html#step-4-set-up-the-environment-variables), run the following commands:
+
+```shell
+cd demos\projects\ESPRESSIF\esp32
+idf.py menuconfig
+```
+
+Under menu item `Azure IoT middleware for FreeRTOS Sample Configuration`, update the following configuration values:
+
+Parameter | Value
+---------|----------
+ `WiFi SSID` | _{Your WiFi SSID}_
+ `WiFi Password` | _{Your WiFi password}_
+
+
+Under menu item `Azure IoT middleware for FreeRTOS Main Task Configuration`, update the following configuration values:
+
+Parameter | Value
+---------|----------
+ `Use PnP in Azure Sample` | Enabled by default. Disable this option to build a simpler sample without Azure Plug-and-Play.
+ `Azure IoT Hub FQDN` | _{Your Azure IoT Hub Host FQDN}_

```

Comment:
This is confusing, if you are using DPS, then you dont need to fill out the FQDN or the device ID.